File-Formats/DocIO/Create-Word-document-in-ASP-NET-Core.md

```diff
@@ -14,15 +14,15 @@ Syncfusion Essential DocIO is a [.NET Core Word library](https://www.syncfusion.
 
 Step 1: Create a new ASP.NET Core Web application project.
 
-![Create ASP.NET Core Web application in Visual Studio](ASP-NET-Core_images/CreateProject.png)
+![Create ASP.NET Core Web application in Visual Studio with File Format DocOI](aspnet-core_images/file-format-docio-aspnet-core-web-application.png)
```

Comment:
Image name should be meaningful. You can change like as below
create-word-in-aspnet-core-web-application
Please check and correct the image name in all places.

Reply:
Updated the changes